providers/powerdns/export_test.go

```diff
@@ -0,0 +1,5 @@
+package powerdns
```

Comment:
I'm not clear about the function of this file.

Reply:
This was a proxy file for non exported functions used for testing. With this file i've exported the functions for the test file